c/Makefile

```diff
@@ -56,9 +56,12 @@ clean-gherkin-languages: ## Remove gherkin-languages.json and any derived files
 	$(MAKE) --directory=src
 	touch $@
 
-clean:
+mostlyclean:
 	rm -rf .compared .built .run acceptance
-	$(MAKE) --directory=src $@
+	$(MAKE) --directory=src clean
```

Comment:
Not every makefile further down has a `mostlyclean`, fixed it like this, but not sure if that is what you intended.